<div class="modal fade signUpErrorModal" id="signUpErrorModal">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-body">

          <div class="signUpError_ttlwrap">
            <h2 class="signUpError_ttl">ユーザー登録に失敗しました</h2>
          </div><!--  .signUpError_ttlwrap -->
          <%= form_for(resource, as: resource_name, url: registration_path(resource_name), remote: true) do |f| %>
            <%= render "users/shared/error_messages", resource: resource %>
          <% end %>
          <div class="signUpError_subtxtwrap">
            <p class="signUpError_subtxt">申し訳ございません。もう一度ユーザー登録を試してみてください。</p><!--  .signUpError_subtxt -->
          </div><!--  .signUpError_subtxtwrap -->
          <div class="signUpError_linkwrap">
            <div class="signUpError_links">
              <!-- コードが読み込めない不具合と戦っていた。どうやらrootページ側のscriptが読み込めないらしい。 -->
              <!-- scriptで再読み込みをさせても治らないため、原因は「rootにきちんと戻れていない」ことにあると考えた。 -->
              <!-- こうしたら治った。けどどうすれば治るのかはまだわからない。わけがわからない…。 -->
              <%= button_to "トップに戻る", root_path, class: "link quit"%>
              <a class="nav-link link entry" id="user-submit-toggler-signuperror" type="button" data-toggle="modal" data-target="#signUpModal"><p><span class="sp">ユーザー登録を</span>やり直す</p></a>
            </div><!--  .signUpError_links -->
          </div><!--  .signUpError_linkwrap -->

      </div>
    </div>
  </div>
</div>
<script type="text/javascript">
  var signupError = document.getElementById("signUpErrorModal");

  window.addEventListener('load',function(){
    signupError.style.display = "block";
    signupError.style.opacity = 1;
    signupError.style.backgroundColor = "rgba(0,0,0, 0.4)";

  });
  // $("#signUpErrorModal").modal();


</script>
